src/birdie/internal/titles.gleam

```diff
@@ -527,9 +532,9 @@ fn try_fold_fields(
 ) -> Result(a, b) {
   use acc, field <- list.try_fold(over: fields, from: acc)
   case field {
-    glance.LabelledField(label: _, item:) | glance.UnlabelledField(item:) ->
-      try_fold_expression(item, acc, fun)
-    glance.ShorthandField(label: _) -> Ok(acc)
+    glance.LabelledField(label: _, label_location: _, item:)
+    | glance.UnlabelledField(item:) -> try_fold_expression(item, acc, fun)
+    glance.ShorthandField(label: _, location: _) -> Ok(acc)
```

Comment:
It appears that you oscillate between using the label accessors and not (as well as doing `_` vs `_x`), so I tried to keep the style consistent with the existing item. Let me know if this was the wrong call.